scholarx/archive/index.html

```diff
@@ -136,10 +136,6 @@ <h6>UNIVERSITY</h6>
           </div>
       </div>
         <div class="col-9">
-            <div class="search-filter">
-                <input id="search" type="text" class="form-control rounded search-bar" placeholder="Search" aria-label="Search" aria-describedby="search-addon" />
-                <button type="button" class="btn btn-outline-primary search-btn">search</button>
```

Comment:
We need to remove only the button @dilankavishka. Keep the search input

Reply:
Done! @anjula-sack